README.es.md

```diff
@@ -344,26 +344,26 @@ Dibuja la pantalla y espera indefinidamente (no usarlo en aplicaciones normales)
 Guarda el archivo de recurso (.pyxres) en el directorio donde se ejecuta el script
```

Comment:
sound and music mean the name of Pyxel instances (class) and can be obtained with the same name APIs. So it should not be translated.